include/battle.h

```diff
@@ -831,6 +831,8 @@ struct BattleStruct
     u8 redCardActivates:1;
     u8 usedEjectItem;
     u8 usedMicleBerry;
+    u8 monCausingSleepClause[NUM_BATTLE_SIDES]; // Stores which pokemon on a given side is causing Sleep Clause to be active as the mon's index in the party
+    u8 sleepClauseEffectExempt; // Stores whether effect should be exempt from triggering Sleep Clause (Effect Spore)
```

Comment:
as I learned from Phallen. This only needs 4 bits
```suggestion
    u8 sleepClauseEffectExempt:4; // Stores whether effect should be exempt from triggering Sleep Clause (Effect Spore)
```